docs/src/pages/components/buttons/buttons.md

```diff
@@ -94,6 +94,12 @@ Here are some examples of customizing the component. You can learn more about th
 
 🎨 If you are looking for inspiration, you can check [MUI Treasury's customization examples](https://mui-treasury.com/styles/button).
 
+## Busy Buttons
+
+The busy buttons can show pending state and disable interactions.
+
+{{"demo": "pages/components/buttons/BusyButtons.js"}}
```

Comment:
What about we create two demos for this feature?

In the first demo, we should keep it minimal, have the inline preview showing, for instance:

<img width="845" alt="Capture d’écran 2020-06-12 à 12 58 27" src="https://user-images.githubusercontent.com/3165635/84496005-783f7b00-acac-11ea-8cc2-1d233b918555.png">

And a second demo to have more cases and showcase the state transition with the toggle?

Reply:
Added